git-backdate

```diff
@@ -122,7 +133,7 @@ def rewrite_history(
             ],
             env=dict(os.environ, GIT_COMMITTER_DATE=timestamp.isoformat()),
         )
-        last_timestamp = timestamp
+        prev_timestamp = timestamp
```

Comment:
So this is where I'm really baffled – not only is `last_timestamp` at least as correct as `prev_timestamp`, it's also two full words rather than an abbreviation. Matter of taste, I suppose, but I don't see how it's relevant to this PR.